pkg/sysched/clientset/v1alpha1/spo_client.go

```diff
@@ -0,0 +1,38 @@
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+)
+
+type SPOV1Alpha1Interface interface {
+	Profiles() ProfileInterface
+}
+
+type SPOV1Alpha1Client struct {
+	RestClient rest.Interface
+}
+
+func NewForConfig(c *rest.Config) (*SPOV1Alpha1Client, error) {
```

Comment:
We don't need this as this should have been abstracted via generic (dynamic) clientset. You can just use the generic clientset in pkg/sysched/sysched.go w/o composing pkg/sysched/clientset at all.

Example: https://github.com/kubernetes-sigs/scheduler-plugins/blob/c752c54e548fdb038cf88a52666b3af1c5b6cee8/pkg/coscheduling/coscheduling.go#L69-L73